polkadot/zombienet_tests/elastic_scaling/assign-core.js

```diff
@@ -1,7 +1,9 @@
-async function run(nodeName, networkInfo, _jsArgs) {
+async function run(nodeName, networkInfo, core) {
```

Comment:
`core` here is an array (`["0"]` or `["1"]`), currently is working because `js` coerce an array of a single element with `Number(["0"])` for example, but could lead to errors in the future. We can change it to  

```suggestion
async function run(nodeName, networkInfo, jsArgs) {
    const core = jsArgs[0];
```

To ensure we are using the correct argument.
Thx!!

Reply:
I tried that initially, but it seems that the jsArgs contained just one value so I assumed this is the way it works. I'll give it one more shot. Does zombienet expand the params separated like this "arg1,arg2,arg3" ?

Reply:
done